### ISSUE TITLE
[Q] Progressbar has border and padding with using "yeti theme".

### ISSUE THREAD

**Issue by Guang1234567** [mkdocs/mkdocs-bootswatch#46]
Hello dev:

I use the `yeti` theme in https://bootswatch.com/yeti/

Then i write 

```html

<div class="progress">
  <div class="progress-bar bg-success" role="progressbar" style="width: 25%" aria-valuenow="25" aria-valuemin="0" aria-valuemax="100"></div>
</div>
<div class="progress">
  <div class="progress-bar bg-info" role="progressbar" style="width: 50%" aria-valuenow="50" aria-valuemin="0" aria-valuemax="100"></div>
</div>
<div class="progress">
  <div class="progress-bar bg-warning" role="progressbar" style="width: 75%" aria-valuenow="75" aria-valuemin="0" aria-valuemax="100"></div>
</div>
<div class="progress">
  <div class="progress-bar bg-danger" role="progressbar" style="width: 100%" aria-valuenow="100" aria-valuemin="0" aria-valuemax="100"></div>
</div>

```

into my markdown file, and it looks like below:

![20180612134944](https://user-images.githubusercontent.com/8240718/41272398-85b1c748-6e47-11e8-94ac-7324610f6c42.png)

not render like below in https://bootswatch.com/yeti/

![20180612134944](https://user-images.githubusercontent.com/8240718/41272487-e5a0960c-6e47-11e8-9525-0624be098019.png)

**Comment by waylan** [comment]
I'm guessing here but I would assume the multi-color progress bars require bootstrap 3. However, we are using Bootstrap 2 for the reasons cited [here](https://github.com/mkdocs/mkdocs/issues/1107#issuecomment-266019907).

**Comment by waylan** [comment]
I'm closing this as resolved in #58.
